blueprints/incredible-squaring/blueprint.json

```diff
@@ -57,9 +31,18 @@
   "request_hook": "None",
   "request_params": [],
   "gadget": {
-    "Wasm": {
-      "runtime": "Wasmtime",
-      "sources": []
+    "Native": {
+      "sources": [
+        {
+          "fetcher": {
+            "Testing": {
+              "cargo_package": "incredible-squaring-blueprint",
+              "cargo_bin": "main",
+              "base_path": "/Users/nologik/commonwealth/gadget/blueprints/incredible-squaring"
```

Comment:
```suggestion
```

Reply:
I'm thinking we should just add blueprint.json to .gitignore since it is auto-generated.